platforms/web/public/locales/en/common.json

```
@@ -28,5 +28,6 @@
   "slide_left": "Slide left",
   "slide_next": "Next slide",
   "slide_previous": "Previous slide",
-  "slide_right": "Slide right"
+  "slide_right": "Slide right",
+  "videoplayer": "Videoplayer"
```

Two words in English:
```suggestion
  "videoplayer": "Video player"
```